lib/firecord/credentials.rb

```diff
@@ -0,0 +1,40 @@
+require 'jwt'
+
+module Firecord
+  class Credentials
+    def initialize
+      @credentials = JSON.parse(File.read("./credentials.json"))
+      @private_key = OpenSSL::PKey::RSA.new @credentials["private_key"]
+    end
+
+    def project_id
+      @credentials['project_id']
+    end
+
+    def generate_jwt_assertion
+      now_seconds = Time.now.to_i
+      payload = {iss: @credentials["client_email"],
+                 aud: @credentials["token_uri"],
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.

[Read more about it here.](http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Style/StringLiterals)